lib/widgets/game_view.dart

```diff
@@ -1,11 +1,28 @@
 import 'package:flutter/material.dart';
+import 'package:tic_tac_toe/widgets/rules_buttons.dart';
 
 import '../grids_page.dart';
 import '../model/boards.dart';
 import '../model/game.dart';
 import '../styles/colors.dart';
 import 'board.dart';
 
+var winner = '';
+
+late final Game game1;
+int inRowValue = 3;
+bool misere = false;
+int boardSize = 3;
+int whoWins = 0;
+
+bool _clickable = true;
+
+bool get clickable => _clickable;
+
+set clickable(bool value) {
+  _clickable = value;
+}
+
```

Comment:
Most of the added items are not needed. Remove them.
Please, add doc comments to needed ones to explain what they mean and how they are used.

Reply:
removed

Reply:
removed